drivers/chown_unix.go

```diff
@@ -49,6 +50,11 @@ func platformLChown(path string, info os.FileInfo, toHost, toContainer *idtools.
 			if err != nil {
 				return fmt.Errorf("%s: lstat(%q): %v", os.Args[0], path, err)
 			}
+			cap, err := system.Lgetxattr(path, "security.capability")
+			if err != nil {
+				return fmt.Errorf("%s: Lgetxattr(%q): %v", os.Args[0], path, err)
```

Comment:
AIUI this will always return an error on non-Linux hosts.

Reply:
... so we either need to check for and ignore `system.ErrNotSupportedPlatform`, or add a note that we're going to have to fix this later.